### PR TITLE
Please increase the maximum of the custom sounds

### PR DESCRIPTION
**Pull request by eried**

### CODE REVIEW

File: src/music.h [hunk 1]
```diff
@@ -29,7 +29,7 @@ enum Music {
 };
 
 #if HAS_EXTENDED_AUDIO
-#define MAX_VOICEMAP_ENTRIES 400 // arbitraty chosen
+#define MAX_VOICEMAP_ENTRIES 500 // arbitraty chosen
```
Comment (PhracturedBlue) [inline src/music.h]: Lint Error:
```
32:  At least two spaces is best between code and comments  
```

Comment (PhracturedBlue) [inline src/music.h]: Lint Error:
```
32:  At least two spaces is best between code and comments  
```

**Comment by eried** [comment]
Thanks! I added some sound fx and noticed that after the internal id 217 my controller was hanging. Limit can be lower, 450 or so, just to have space for around 60-100 extra custom sounds

https://github.com/eried/Research/tree/master/JumperT8SG/soundfx

**Comment by TheRealMoeder** [comment]
Development around deviation is rather quiet, we might as well merge this. If we ever run into ram issues, we can rethink things.

**Pull request merged**